GearSpawner/GearSpawnReader.cs

```diff
@@ -24,19 +24,8 @@ internal static class GearSpawnReader
 		@"(?:\W+r\s*=\s*(" + VECTOR + "))?" +
 		@"(?:\W+\s*c\s*=\s*(" + NUMBER + "))?$");
```

Comment:
Mark all 5 of these regexes as compiled. It's an option in the second parameter.